.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+updates:
+  # Monitor Github Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "10:00"
+    commit-message:
+      prefix: "chore"
+      include: "scope"
```

Comment:
IMO is better to use a specific `prefix` for GHA commit messages such as `"gha"`

Reply:
We can do that, but I thought to not diverge from the general practice - ref. https://www.conventionalcommits.org/en/v1.0.0/#summary 

In that sense, the spec more or less combines everything dependency related under `build`. 

Do we prefer to have everything under `chore`(or `build`) or have separate ones like `gha`, `pip`, `pip dev`, and so forth?

Reply:
Got it!

Reply:
Settled to `build-dev` for development related dependencies and `build` for everything else 😃